DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogicND.cs

```diff
@@ -972,7 +1042,7 @@ private static IEnumerable<DevicePropertyMetadata> GetDevicePropertyConfiguratio
             };
         }
 
-        private async Task ValidateDevice(dynamic device)
+        private async Task ValidateDevice(DeviceND device)
```

Comment:
why not ValidateDeviceND ? have we removed dynamics from every reference of ValidateDevice ?